src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessClientExtensions.cs

```diff
@@ -36,6 +37,24 @@ public static class ExceptionlessClientExtensions {
             return null;
         }
 
+        internal static string[] GetTags(this LogEvent log)
```

Comment:
I think the best spot to set this would be in the ExceptionlessSink around line 142, we are already iterating over the log event properties and then we would just need to cast and add. I'd also maybe just look for IEnumerable for the cast so it works with any collection type.

Reply:
Thanks @niemyjski  yeah that makes a lot more sense where you suggested. I've pushed the change. I've left case "Tags": in the case for now as I don't see an existing Constant for this. Let me know your thoughts?

Happy to update readme, will leave this unresolved until I get there